src/middleware/authentication/middleware.ts

```diff
@@ -1,9 +1,77 @@
 import type { NextFunction, Request, Response } from "express";
+import { SignJWT } from "jose";
+import axios from "axios";
+// biome-ignore lint/correctness/noNodejsModules: we need the node crypto module
+import crypto from "node:crypto";
 
-export function authenticationMiddleware(
+export async function authenticationMiddleware(
   _request: Request,
```

Comment:
You can remove the "_" before request and response. They were added to comply with Biome detecting unused variables.